### DISCUSSION TITLE
Aqara home? Mi home? Both? None?

### DISCUSSION THREAD

**Discussion by Tiwas** [PiotrMachowski/Xiaomi-cloud-tokens-extractor#62] [Q&A]
I tried this tool today to get the token needed to set this up on my Homey. I had everything set up with the Aqara Home app, but got nothing when running this tool. I then reset everything and set up the hub in Mi Home, and still got nothing. My roller shade driver e1 wasn't supported anyway, so no big deal. However - is there no way to get the token when using either of these apps?

I enter my user ID (international phone number, space between the international number and the rest) and password. I don't select a server, so it checks them all. This is the reply I get, both having the hub set up in Mi Home or Aqara Home
![image](https://user-images.githubusercontent.com/17053402/157734864-d758723b-b8f7-49e5-9d5a-e76e5946ff95.png)

Any help would be appreciated :)

**Comment by PiotrMachowski** [comment]
It works with Mi Home/Xiaomi Home. You have to use your main account though (not shared one)

**Reply by Tiwas** [reply]
Thanks, but I didn't get anything when they were in Mi Home either. I'm using the main account (haven't shared it). Hmmm...Guess I'll have to figure out how to get the roller shade driver into Mi Home and try again.
